JWBestPracticeApps/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -29,8 +29,13 @@ class FeedTableViewController: UITableViewController {
         
         // Populate the feed array with video players
         for itemInfo in feedInfo {
-            if let config = JWConfig.init(contentUrl: itemInfo["url"]),
-                let player = JWPlayerController.init(config: config) {
+            guard let url = itemInfo["url"] else {
+                continue
+            }
+            
+            let config = JWConfig.init(contentUrl: url)
+            
+            if let player = JWPlayerController.init(config: config) {
```

Comment:
This is probably just a nickpick, but I don't think the Swift conventions calls for using init for the constructors. Why not just JWPlayerController(config: config)? Same for all other constructors.

Reply:
You are right, and I'll fix this. I believe it's a case of me following the convention set by the code I replaced.